kairos-cli/src/commands/transfer.rs

```diff
@@ -0,0 +1,31 @@
+use crate::common::args::{AmountArg, PrivateKeyPathArg};
+use crate::crypto::error::CryptoError;
+use crate::crypto::public_key::CasperPublicKey;
+use crate::crypto::signer::CasperSigner;
+use crate::error::CliError;
+use crate::utils::parse_hex_string;
+
+use clap::Parser;
+
+#[derive(Parser)]
+pub struct Args {
+    #[arg(long, short, value_name = "PUBLIC_KEY", value_parser = parse_hex_string)]
+    recipient: ::std::vec::Vec<u8>, // Absolute path is required here - see https://github.com/clap-rs/clap/issues/4626#issue-1528622454.
+    #[clap(flatten)]
+    amount: AmountArg,
+    #[clap(flatten)]
+    private_key_path: PrivateKeyPathArg,
+}
+
+pub fn run(args: Args) -> Result<String, CliError> {
+    let _recipient = CasperPublicKey::from_bytes(args.recipient.as_ref())?;
```

Comment:
So... do we have a test that public keys on https://cspr.live/ parse using this?

Reply:
Tests added in https://github.com/cspr-rad/kairos/pull/17/commits/3775a71005d2f99a8e9dbe9d158ea16c052abab2.